src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,19 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+
+        Machine[] machines = new Machine[]{
+                new Bulldozer(), new Truck(), new Excavator()
+        };
+
+        for (Machine machine : machines) {
+
+            machine.doWork();
+            machine.stopWork();
+
```

Comment:
Remove the redundant empty line after the loop block to follow the checklist's best practices.